dev/tests/integration/testsuite/Magento/Framework/Code/GeneratorTest.php

```diff
@@ -172,6 +173,74 @@ public function testGenerateClassExtensionAttributesInterfaceFactoryWithNamespac
         $this->assertEquals($expectedContent, $content);
     }
 
+    public function testGenerateClassProxyWithNamespace71()
+    {
+        if (!$this->shouldRun71()) {
+            $this->assertTrue(true);
```

Comment:
`markTestSkipped` must be used in case of lower PHP version instead, please fix all occurrences.